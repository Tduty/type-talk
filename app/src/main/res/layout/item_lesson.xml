<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="wrap_content">

    <androidx.cardview.widget.CardView
        android:layout_width="match_parent"
        android:layout_height="240dp"
        android:layout_marginBottom="8dp"
        android:layout_marginTop="4dp"
        android:layout_marginStart="10dp"
        android:layout_marginEnd="10dp"
        app:cardPreventCornerOverlap="true"
        app:cardBackgroundColor="@color/white"
        app:cardCornerRadius="10dp"
        app:cardElevation="2dp">

        <LinearLayout
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:orientation="vertical">

            <androidx.constraintlayout.widget.ConstraintLayout
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:background="@drawable/bg_lesson_from_above"
                android:orientation="vertical">

                <androidx.appcompat.widget.AppCompatImageView
                    android:id="@+id/iv_lesson_ic"
                    android:layout_width="96dp"
                    android:layout_height="96dp"
                    android:layout_margin="10dp"
                    app:layout_constraintBottom_toBottomOf="parent"
                    app:layout_constraintStart_toStartOf="parent"
                    app:layout_constraintTop_toTopOf="parent"
                    app:srcCompat="@drawable/ic_teacher" />

                <TextView
                    android:id="@+id/tv_lesson_nmb"
                    android:layout_width="0dp"
                    android:layout_height="wrap_content"
                    android:layout_marginStart="10dp"
                    android:layout_marginEnd="10dp"
                    android:ellipsize="end"
                    android:lines="1"
                    android:textColor="@color/tundora"
                    android:textSize="20sp"
                    app:layout_constraintEnd_toStartOf="@+id/ll_lesson_status"
                    app:layout_constraintStart_toEndOf="@+id/iv_lesson_ic"
                    app:layout_constraintTop_toTopOf="@+id/iv_lesson_ic"
                    tools:text="Lesson 1" />

                <LinearLayout
                    android:id="@+id/ll_lesson_status"
                    android:layout_width="wrap_content"
                    android:layout_height="wrap_content"
                    android:layout_marginTop="10dp"
                    android:layout_marginEnd="10dp"
                    android:gravity="center"
                    android:orientation="horizontal"
                    app:layout_constraintEnd_toEndOf="parent"
                    app:layout_constraintTop_toTopOf="parent">

                    <androidx.appcompat.widget.AppCompatImageView
                        android:id="@+id/iv_lesson_checkbox"
                        android:layout_width="26dp"
                        android:layout_height="26dp"
                        app:srcCompat="@drawable/ic_checkbox_empty" />

                    <TextView
                        android:id="@+id/tv_lesson_status"
                        android:layout_width="wrap_content"
                        android:layout_height="wrap_content"
                        android:layout_marginStart="3dp"
                        android:textColor="@color/tundora"
                        android:textSize="16sp"
                        tools:text="Complete" />

                </LinearLayout>

                <TextView
                    android:id="@+id/tv_lesson_title"
                    android:layout_width="0dp"
                    android:layout_height="wrap_content"
                    android:layout_marginStart="10dp"
                    android:layout_marginTop="5dp"
                    android:layout_marginEnd="10dp"
                    android:ellipsize="end"
                    android:lines="1"
                    android:textColor="@color/black"
                    android:textSize="32sp"
                    android:textStyle="bold"
                    app:layout_constraintEnd_toEndOf="parent"
                    app:layout_constraintStart_toEndOf="@+id/iv_lesson_ic"
                    app:layout_constraintTop_toBottomOf="@+id/tv_lesson_nmb"
                    tools:text="Weather" />

            </androidx.constraintlayout.widget.ConstraintLayout>

            <androidx.constraintlayout.widget.ConstraintLayout
                android:layout_width="match_parent"
                android:layout_height="wrap_content">

                <TextView
                    android:id="@+id/tv_lesson_content"
                    android:layout_width="0dp"
                    android:layout_height="0dp"
                    android:layout_margin="10dp"
                    android:ellipsize="end"
                    android:maxLines="6"
                    android:textColor="@color/black"
                    android:textSize="14sp"
                    app:layout_constraintBottom_toBottomOf="parent"
                    app:layout_constraintEnd_toStartOf="@+id/ll_lesson_expected"
                    app:layout_constraintStart_toStartOf="parent"
                    app:layout_constraintTop_toTopOf="parent"
                    tools:text="Weather " />

                <LinearLayout
                    android:id="@+id/ll_lesson_expected"
                    android:layout_width="0dp"
                    android:layout_height="wrap_content"
                    android:layout_margin="10dp"
                    android:orientation="vertical"
                    app:layout_constraintBottom_toBottomOf="parent"
                    app:layout_constraintEnd_toEndOf="parent"
                    app:layout_constraintTop_toTopOf="parent">

                    <TextView
                        android:layout_width="wrap_content"
                        android:layout_height="wrap_content"
                        android:textColor="@color/black"
                        android:textSize="16sp"
                        android:textStyle="bold"
                        tools:text="Expected:" />

                    <include
                        android:id="@+id/item_lesson_expected_one"
                        layout="@layout/item_lesson_expected" />

                    <include
                        android:id="@+id/item_lesson_expected_two"
                        layout="@layout/item_lesson_expected" />

                    <include
                        android:id="@+id/item_lesson_expected_three"
                        layout="@layout/item_lesson_expected" />

                </LinearLayout>

            </androidx.constraintlayout.widget.ConstraintLayout>

        </LinearLayout>

    </androidx.cardview.widget.CardView>

</RelativeLayout>