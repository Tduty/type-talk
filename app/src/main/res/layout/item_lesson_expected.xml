<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    android:layout_marginTop="5dp"
    android:gravity="center"
    android:orientation="horizontal">

    <androidx.appcompat.widget.AppCompatImageView
        android:id="@+id/iv_expected_ic"
        android:layout_width="23dp"
        android:layout_height="23dp"
        app:srcCompat="@drawable/ic_lesson_expected_dialoges" />

    <TextView
        android:id="@+id/tv_lesson_expected"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_marginStart="6dp"
        android:textColor="@color/tundora"
        android:textSize="16sp"
        tools:text="Dialoges" />

</LinearLayout>